wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,114 @@
+# What is Restriction mode exactly?
+
+
+
+A restriction is a timeout from the community forced upon accounts that exhibit abnormal, suspicious, or rule-breaking behaviour.
+
+
+
+When an account is restricted, it is unable to interact with the community until the restriction is lifted. Restricted players can still continue playing and submitting scores, but their profile will be invisible to others.
+
+
+
+The following features are disabled for restricted accounts:
+
+- in-game chat
+
+- private messaging on the forums
+
+- posting on forums
+
+- updating maps or profile details
+
+- store purchases
+
+
+
+# My account has been restricted! What do I do?
+
+
+
+If your account has been restricted, don't panic. Please follow these steps:
+
+
+- **Wait 24 hours.** Some restrictions are the results of automated processes. As we check every new restriction daily, please wait 24 hours - if your restriction was erroneous, it will be lifted within that time.
+
+- **Contact our support team.** If your account remains restricted after that time, please send an email to accounts@ppy.sh to ask about the reasons for it. We'll need at the very least to know your username to be able to check! Remember to send the email from the e-mail address attached to your account so that we can confirm you are the owner.
```

Comment:
keep `email`/`e-mail` consistent?
```suggestion
- **Contact our support team.** If your account remains restricted after that time, please send an email to accounts@ppy.sh to ask about the reasons for it. We'll need at the very least to know your username to be able to check! Remember to send the email from the email address attached to your account so that we can confirm you are the owner.
```